iphone/Classes/TiUIButton.m

```diff
@@ -412,26 +412,9 @@ -(void)setVerticalAlign_:(id)align
 -(void)setTextAlign_:(id)align
 {
     button = [self button];
-    NSTextAlignment alignment = NSTextAlignmentNatural;
```

Comment:
Note that this actually does change behavior here. If the user specified null/undefined/ or a bad int value, we'd default to NSTextAlignmentNatural, while we can no longer specify any value to get that setting now. We may want to expose a "natural" constant value as well?